### ISSUE TITLE
[Question] SVT-AV1 1.7.0

### ISSUE THREAD

**Issue by UpscaleAnon** [n00mkrad/nmkoder#49]
I'm just curious, do you have plans to update now that SVT-AV1 1.7.0 is a thing? Considering the MASSIVE upgrade, I'd hope to see it soon. I'd love to give it a try and there doesn't seem to be any ffmpeg windows build with it yet

**Comment by jkkma** [comment]
n00mkrad is busy making the ai dream happen, in the meantime hop on the AV1 discord theres a link there with fully updated nmkoder, of course with svt 1.7 too

**Comment by UpscaleAnon** [comment]
> n00mkrad is busy making the ai dream happen, in the meantime hop on the AV1 discord theres a link there with fully updated nmkoder, of course with svt 1.7 too

No, I'm banned from that discord server run by trannies and retards

**Comment by ArakiSatoshi** [comment]
Well, yeah, it's pretty much abandoned. But as @jkkma said, the community still "supports" it ~~(i.e. shares links on Discord for you to click on and execute unknown .exe's on your main PC)~~.

https://www.mediafire.com/file/j1z9zi18ra42od1/Nmkoder_FFWAE_vapoursynth_updated.7z/file

From Discord, credits to the one and only... _Deleted User_:
https://discord.com/channels/587033243702788123/1042536514783023124/1144430045427929180

**Comment by UpscaleAnon** [comment]
Man, a year later, SVT-AV1 2.0.0 and 2.1.0 has come out with massive improvements, aswell as psy version, but N00mkrad still slacking.
Crazy how there's 434 patrons paying him money when he's not even doing anything.

**Comment by ArakiSatoshi** [comment]
@UpscaleAnon He's busy with AI-related stuff, though, for example, the free version of his Flowframes app is also heavily outdated.

At this point, `nmkoder` needs to be archived.
